src/main/java/dev/tourmi/svmm/SVMM.java

```diff
@@ -32,11 +31,11 @@ public class SVMM
 
     private final VeinMiner veinMiner;
 
-    public SVMM()
+    public SVMM(FMLJavaModLoadingContext modLoadingContext)
     {
         veinMiner = new VeinMiner();
 
-        ModLoadingContext.get().registerConfig(ModConfig.Type.SERVER, SVMMConfig.SPEC, "svmm-config.toml");
+        modLoadingContext.registerConfig(ModConfig.Type.SERVER, SVMMConfig.SPEC, "svmm-config.toml");
```

Comment:
This change needs to be reverted. I know `ModLoadingContext.get()` is deprecated, but it is needed for 1.21, since FMLJavaModLoadingContext does not extend ModLoadingContext in forge `51`